rayshader_cheatsheet_3dviz.Rmd.Rmd

```diff
@@ -0,0 +1,217 @@
+---
+title: "Community Contribution"
+author: "Yu Song Ng"
+date: "2022-11-08"
+output: html_document
+---
+
+```{r setup, include=FALSE}
```

Comment:
use {r, include=FALSE} instead of {r setup, include=FALSE}.

Reply:
Amended this

Reply:
@nys98, your file says Rmd.Rmd. Can you please recommit with .Rmd